src/wireguard/Makefile

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Nit: should we update the license here?

Reply:
As discussed, we'll defer the license headers to later